CHANGELOG.md

```diff
@@ -1,25 +1,26 @@
-# 17.3.0
+# 17.3.0 - 24.05.2024
 ## Features
-- IFS-2876: Aria-Label-Attribute wurde zur Verbesserung der Barrierefreiheit verwendet
+- IFS-2876: Aria-Label-Attribute wurden zur Verbesserung der Barrierefreiheit an Icons und Buttons in den Komponenten `isy-hauptfenster`, `isy-wizard` und `isyInputChar` hinzugefügt
 - IFS-3684: Das Basis-Theme _Bootstrap4 Light Blue_ wurde als SCSS in das _IsyFact-Theme_ integriert. Eine Live-Bearbeitung der Variablen ist nun möglich
-- IFS-3118: Eingabefeld für ungewisses Datum `isy-incomplete-date` kann nun Datumswerte in ISO 8601 für die Datenübertragung ausgeben. Validator für ungewissen Datumswert in ISO 8601 wurde zur Bibliothek hinzugefügt
+- IFS-3118: Das Eingabefeld für ungewisse Datums `isy-incomplete-date` kann nun Datumswerte im ISO 8601 Format für die Datenübertragung ausgeben. Ein Validator für ungewisse Datumswerte im ISO 8601 Format wurde zur Bibliothek hinzugefügt
 - IFS-3644: Im `isy-form-wrapper` wurde das Label der Formularfelder von Float-Label auf Static-Label geändert
-- IFS-3645: Infield Top-Aligned Label wurde zum `isy-form-wrapper` hinzugefügt
+- IFS-3645: Infield Top-Aligned Label (IFTA Label) wurde zum `isy-form-wrapper` als optionale Labelvariante hinzugefügt
 - IFS-3605: Icons zum Öffnen und Schließen der Seitenleisten sind in einfache Chevrons verändert worden
```

Comment:
Das ist nur in der Demo App oder? würde ich dann entpsrechend verhieben

Reply:
Nein, das ist als Property in der Form Wrapper Komponente eingebaut. Als Default aber als False und zur Zeit auch nicht im Einsatz in der DemoApp.